parla/parray/coherence.py

```diff
@@ -0,0 +1,241 @@
+from __future__ import annotations # To support self references in type checking, must be first line

```

Comment:
Don't need this comment.

Reply:
This is for type hint when refer to itself